opencbm/sys/freebsd/.gitignore

```diff
@@ -0,0 +1,9 @@
+*
+*/
+!LINUX
+!LINUX/*
+!opencbm.c
+!opencbm.4
+!Makefile
+!.gitignore
+
```

Comment:
Is the .gitignore file really needed? Personally, I prefer to have an almost clean .gitignore, but have all the unnecessary files removed with "make mrproper".

Reply:
Spiro, you can remove it, it is just a suggestion: I personally prefer to never have any files appear in `git status` that shouldn't be added to git. I know this is a matter of taste, so if you don't like it, get rid of it :)

Reply:
If you prefer, I can rewrite the history of this PR without the .gitignore? I mean, for a clean commit? Or maybe you like my argumentation? ;)

Reply:
It's ok for me.
The reason why I do not like the .gitignore: I do not see directly if the "mrproper" target works as expected.
But as it is "only" in the FreeBSD part, I could live with it, as I do not/cannot touch this.